src/main/resources/helpTemplates/common.html

```diff
@@ -0,0 +1,53 @@
+<#--
+        This file contains part of the theming used to present Barclay docs on a website. Styling is separated
+        out, so pages will be minimalistic html unless replacement styling is provided.
+        -->
+
+    <#macro footerInfo>
+        <hr>
+        <p><a href='#top'><i class='fa fa-chevron-up'></i> Return to top</a></p>
```

Comment:
FYI some elements here are very specific to libraries we use in the GATK website, like `i class='fa fa-...'` and `class="nav nav-pills nav-stacked" id="sidenav"`. I don't mind since it's what we use, but if you're trying to be as generic as possible, would you want to move this out entirely? I could propose a completely generic template if that helps.